setup.py

```diff
@@ -12,7 +12,9 @@
     packages=find_packages(),
     include_package_data=True,
     install_requires=[
-        'Twisted>=13.1.0'
+      'klein',
+      'treq',
+      'Twisted>=13.1.0'
```

Comment:
We should probably indent these lines with 4 spaces relative to `install_requires` instead of 2.